server/middlewares/controllers/bookControllers.js

```diff
@@ -0,0 +1,30 @@
+require("dotenv").config();
+
+// const Book = require('../../models/Book');
+const knexfile = require("../../../knexfile")[process.env.NODE_ENV];
+const knex = require("knex")(knexfile);
+
+class BookController {
+  static async addBooks(req, res) {
+    const { title, coverType, description, isbn, publisher, year } = req.body;
+
+    // await Book
```

Comment:
Why do we have this comments ? This looks like the right way to go. Isn't it working ?

Reply:
Yes, it doesn't work when I try to implement it that way. I would appreciate if you could help look through it